### DISCUSSION TITLE
Error while executing build.sh

### DISCUSSION THREAD

**Discussion by tariq-ansari** [Sunbird-Ed/Community#17] [Installation] [answered]
Hi @sharathkashyap, Following your instructions we were able to complete the npm installation step. 
However, once we are executing build.sh, we are running into the following error.
<img width="1792" alt="Screenshot 2022-06-02 at 1 23 35 PM" src="https://user-images.githubusercontent.com/68503506/171582787-53f2e799-c9ed-4af4-bd4b-4adb132e280d.png">

I tried removing and adding android as a platform, using:
_cordova platform remove android
cordova platform add android_
but this doesn't seems to work and we are getting this error again.

**Comment by sharathkashyap** [comment]
@tariq-ansari The Root Cause of the Failure/Error is not visible in the screenshot. Is it possible to share the entire log of the build. It will be helpful to debug the issue.

**Reply by tariq-ansari** [reply]
@sharathkashyap It appears that it was because of some error in python version for mac. 
This error is now resolved.

**Comment by tariq-ansari** [comment, marked as answer]
@sharathkashyap We are getting the following error while executing build.sh
<img width="1792" alt="Screenshot 2022-06-03 at 12 14 56 PM" src="https://user-images.githubusercontent.com/68503506/171801903-70c252f1-48cf-49be-a76d-7f7754f7e9a3.png">

**Reply by sharathkashyap** [reply]
@tariq-ansari The issue was to comment the build-extras.gradle for development and production flavours as the build was generated for staging. Root Cause of the issue was that property did not exist in your sunbird.properties file.
